skrub/_table_vectorizer.py

```diff
@@ -449,6 +472,25 @@ def _clone_transformers(self):
         else:
             self.datetime_transformer_ = self.datetime_transformer
 
+        if self.column_specific_transformers is None:
+            self.column_specific_transformers_ = []
+        else:
+            if len(self.column_specific_transformers[0]) == 2:
+                # Unnamed assignments, transform to named
+                named_column_specific_transformers = _get_transformer_list(
+                    self.column_specific_transformers
+                )
+            elif len(self.column_specific_transformers[0]) == 3:
+                # Named assignments
+                named_column_specific_transformers = self.column_specific_transformers
```

Comment:
Don't we need an "else" that raises a clean error message?

Reply:
Yes, it's an oversight!